Technomancy/app.ts

```diff
@@ -2869,6 +2869,10 @@ function load() {
 }
 
 function save_to_clip() { /* Put save data in clipboard. Copied from Stack Overflow :) */
+    alert("STUB!")
```

Comment:
Any reason this needs to have functionality removed?

Reply:
Because "Yes".

Reverted because it makes more sense than stubbing it (I don't remember why I did it).